src/shared/utils.ts

```diff
@@ -35,12 +35,15 @@ export function findNetStandardTargetFramework(tfmShortNames: string[]): string
     return tfmShortNames.find((tf) => tf.startsWith('netstandard'));
 }
 
-export function isWebProject(projectPath: string): boolean {
-    const projectFileText = fs.readFileSync(projectPath, 'utf8');
+export function isWebProject(projectPath: string): [boolean, boolean] {
```

Comment:
We now have two ways to detect a Blazor project. Should a project that is a Blazor project always have both? Or is one of them just going to return true for a subset of blazor projects?